submissions/Harnytskyi/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,635 @@
+                            <label>
+                                <input id="cb1" type="checkbox" class="check_input">9
+                            </label>
+                        </div>
+                        <div class="block_item sold_place">
+                            <label>
+                                <input id="cb1" type="checkbox" class="check_input">10
+                            </label>
+                        </div>
+                        <div class="block_item cinema_row">10</div>
+                        <div class="block_item"></div>
+                        <div class="block_item"></div>
+                        <div class="block_item sold_place">
+                            <label>
+                                <input id="cb1" type="checkbox" class="check_input">3
+                            </label>
+                        </div>
+                        <div class="block_item sold_place">
+                            <label>
+                                <input id="cb1" type="checkbox" class="check_input">4
+                            </label>
+                        </div>
+                        <div class="block_item sold_place">
+                            <label>
+                                <input id="cb1" type="checkbox" class="check_input">5
+                            </label>
+                        </div>
+                        <div class="block_item sold_place">
+                            <label>
+                                <input id="cb1" type="checkbox" class="check_input">6
+                            </label>
+                        </div>
+                        <div class="block_item standart_place">
+                            <label>
+                                <input id="cb1" type="checkbox" class="check_input">
+                                <div class="content">7</div>
+
+                            </label>
+                        </div>
+                        <div class="block_item standart_place">
+                            <label>
+                                <input id="cb1" type="checkbox" class="check_input">
+                                <div class="content">8</div>
+                            </label>
+                        </div>
+                        <div class="block_item"></div>
+                        <div class="block_item"></div>
+                    </div>
+                </div>
+                <div class="cost">
+                    <div class="showtimes_propery">
+                        <div class="showtimes_property_header">Movie: </div>
+                        <div class="showtimes_property_value"> Ferris Bueller's day off</div>
+                    </div>
+                    <div class="showtimes_propery">
+                        <div class="showtimes_property_header">Time: </div>
+                        <div class="showtimes_property_value"> April 3, 4:20</div>
```

Comment:
For date/time use tag `time`. [Read more.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)